Example/ViewController+FreeDrive.swift

```diff
@@ -43,6 +43,7 @@ extension ViewController {
         
         if let location = notification.userInfo?[PassiveLocationDataSource.NotificationUserInfoKey.locationKey] as? CLLocation {
             trackStyledFeature.lineString.coordinates.append(contentsOf: [location.coordinate])
+            navigationMapView.updateUserCourseView(location)
```

Comment:
`updateUserCourseView(_:animated:)` should probably be renamed at this point, since it affects the user location indicator even if it’s a location layer instead of a view. Consider `moveUserLocation(to:animated:)`, which also clarifies the relationship between the method and its first argument.

Reply:
Thanks for the suggestion, I'll update the name.